api/envoy/config/route/v3/route_components.proto

```diff
@@ -760,6 +760,30 @@ message RouteAction {
     ConnectConfig connect_config = 3;
   }
 
+  message MaxStreamDuration {
```

Comment:
This should be marked with `[#not-implemented-hide:]`, since it's not yet implemented in Envoy.